Pcap++/src/PcapLiveDevice.cpp

```diff
@@ -1011,15 +1011,15 @@ namespace pcpp
 			return;
 		}
 
-		uint8_t buf[len];
+		std::vector<uint8_t> buf(len);
```

Comment:
Tbh, since `len` is a non-constexpr variable I have... questions about how this compiled before...

Reply:
This is a VLA allocated on stack